qa/suites/fs/volumes/tasks/volumes/test/fwdauth.yaml

```
@@ -0,0 +1,8 @@
+tasks:
+- exec:
+    mon.a:
+      - ceph fs set cephfs balance_automate false
+- cephfs_test_runner:
+    fail_on_skip: false
+    modules:
+      - tasks.cephfs.test_volumes.TestForwardAuth
```

```suggestion
      - tasks.cephfs.test_misc.TestForwardAuth
```

? (I think you meant to move this yaml.)

---

Yeah, fixed it. I will run to test it today.